load-package.php

```diff
@@ -6,14 +6,16 @@
  * @package CoCart Products API Package
  */
 
+use CoCart\ProductsAPI\Package;
```

Comment:
Importing a class does not autoload it.

Reply:
@szepeviktor The class is already auto loaded with the packager. The check is there encase auto loader failed.

Reply:
I've commented this because there is a myth that importing a class loads it.